requirements.txt

```diff
@@ -7,4 +7,8 @@ litellm==1.42.1
 Pillow>=10.1
 beautifulsoup4>=4.12
 fastapi
-uvicorn
\ No newline at end of file
+uvicorn
+chromadb=0.5.20
+llama-index-core=0.12.2
+llama-index-vector-stores-chroma=0.4.0
+llama-index-embeddings-openai=0.3.1
```

Comment:
chromadb==0.5.20
llama-index-core==0.12.2
llama-index-vector-stores-chroma==0.4.0
llama-index-embeddings-openai==0.3.1
should be == instead of =